test/parallel/test-process-redirect-warnings-env.js

```diff
@@ -6,14 +6,15 @@
 // opened and the contents are validated
 
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 const fs = require('fs');
 const fork = require('child_process').fork;
 const path = require('path');
 const assert = require('assert');
 
 common.refreshTmpDir();
 
-const warnmod = require.resolve(`${common.fixturesDir}/warnings.js`);
+const warnmod = require.resolve(fixtures.path('/warnings.js'));
```

Comment:
This could be just `fixtures.path('warnings.js')`.

Reply:
Fixing this on landing